### PR TITLE
SmritiGupta237 - Explain the difference between escape() and unescape() functions

### PR DESCRIPTION
**Pull request by smriti237**

### CODE REVIEW

File: javascript/answers/Smriti-Gupta-q9.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        gtag('config', 'UA-157837769-1');
+        </script>
+        
+        <title>
+            JavaScript | InterQues | Coding Ninjas
+        </title>
+
+        <link rel="shortcut icon" type="image/png" href="../../assets/favicon.png"/>
+
+        <meta name="description" content="JavScript interview questions">
```
Comment (dakshkhetan) [inline javascript/answers/Smriti-Gupta-q9.html]: Change the content="" and add the question title into it. (Look at others' answer file for your reference)

**Review by dakshkhetan — CHANGES_REQUESTED** [review]
Take a fresh pull (from upstream master) and make the required changes.

**Comment by dakshkhetan** [comment]
@smriti237  Any updates? This PR will be closed in 24 hours if no changes are done.

**Comment by smriti237** [comment]
Just changing it

**Comment by smriti237** [comment]
I have made the required changes.

File: javascript/index.html [hunk 2]
```diff
@@ -271,6 +271,15 @@ <h4>
                 </a>
             </li>
 
+             <li>
+                <a href="./answers/Smriti-Gupta-q9.html">
+                    <h4>
+                        Explain the difference between escape() and unescape() functions?
+                    </h4>
+
```
Comment (dakshkhetan) [inline javascript/index.html]: You haven't closed the 'a' tag.

**Review by dakshkhetan — CHANGES_REQUESTED** [review]

File: javascript/index.html (hunk 2 above)
Comment (arpansac) [reply javascript/index.html]: @smriti237 please resolve this ASAP

**Review by arpansac — DISMISSED** [review]

File: javascript/index.html (hunk 2 above)
Comment (smriti237) [reply javascript/index.html]: Added the 'a' tag. I don't know how it vanished. I clearly checked it before pushing. Please review again..

**Review by dakshkhetan — DISMISSED** [review]

**Review by dakshkhetan — APPROVED** [review]

**Pull request merged**